tests/test_htuse.py

```diff
@@ -2,7 +2,7 @@
 from numpy.testing import assert_array_almost_equal
 import pandas as pd
 from pandas.testing import assert_series_equal
-import pytest
+import pytest, pytest_cov
 
```

Comment:
```suggestion
import pytest
```
`pytest_cov` is unused